### PR TITLE
changed telemetry mode baseline and v2-stats-nullvm to no_istio and i…

### PR DESCRIPTION
**Pull request by Shuayb1**
…stio_with_stat respectively

### CODE REVIEW

**Review by howardjohn — APPROVED** [review]

File: perf/benchmark/runner/fortio.py [hunk 1]
```diff
@@ -209,7 +208,7 @@ def sync_fortio(url, table, selector=None, promUrl="", csv=None, csv_output="",
 
     if table:
         return write_table(table, datafile)
-
+    os.system("rm -rf {temp_dir_path}".format(temp_dir_path=temp_dir_path))
```
Comment (AndreaM12345) [inline perf/benchmark/runner/fortio.py:211]: should this be under a flag so its not always deleting?

Comment (AndreaM12345) [reply perf/benchmark/runner/fortio.py:211]: talked with Shuayb about some options on what he was trying to achieve with the temp directory removal.

**Comment by ericvn** [comment]
@Shuayb1 You need to run `make gen` locally in your branch and then commit that change. You also need to accept the CLA.

**Pull request closed without merging**